src/ipahealthcheck/dogtag/ca.py

```diff
@@ -102,10 +103,40 @@ def check(self):
             logger.debug('CA is not configured, skipping connectivity check')
             return
 
+        config = api.Command.config_show()
+        subject_base = config['result']['ipacertificatesubjectbase'][0]
```

Comment:
When the server is installed with `--ca-subject`, the CA subject can be different from `CN=Certificate Authority,$BASE`.
The value can be found using `get_ca_subject()` from ipalib/install/certstore.py or read directly from `cn=ipa,cn=cas,cn=ca,$BASE`, in the attribute `ipaCaSubjectDN`.

Reply:
It's true but both of these methods return the Subject and not the base (well, it's there implicitly). I could use string substitution, or the DN module, to drop the RDN I guess. An API call is more expensive than LDAP but gives me the value I need.